env/includes/settings/tk-multi-workfiles2.yml

```diff
@@ -168,6 +168,55 @@ settings.tk-multi-workfiles2.hiero:
 
 # ---- Houdini
 
+# project
+settings.tk-multi-workfiles2.houdini:
+  entities:
+  - caption: Assets
+    entity_type: Asset
+    hierarchy: [sg_asset_type, code]
+    filters:
+    sub_hierarchy:
+      entity_type: Task
+      filters:
+      link_field: entity
+      hierarchy: [step]
+  - caption: Shots
+    entity_type: Shot
+    filters:
+    hierarchy: [sg_sequence, code]
+    sub_hierarchy:
+      entity_type: Task
+      filters:
+      link_field: entity
+      hierarchy: [step]
+  ui_config_hook: "{self}/ui_config.py:{self}/ui_config_tk-houdini.py"
```

Comment:
I could not find any reference `ui_config_hook`. Where is it defined?

Reply:
Also, is there any engine version requirement? This new hook is not yet available in workfile2. Is it possible that this config runs with an older version or workfile2? If so, what would happen?

Reply:
> I could not find any reference ui_config_hook. Where is it defined?

I should have included this info in the PR, it is hard a little hard to track:  so the `ui_config_hook` is defined in the tk-multi-workfiles2 [info.yml](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/c82875b482a3e1f4d126f4646744d82231a85e16/info.yml#L116-L120) and is used [here](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/c82875b482a3e1f4d126f4646744d82231a85e16/python/tk_multi_workfiles/file_model.py#L500-L501).

> Also, is there any engine version requirement? This new hook is not yet available in workfile2. Is it possible that this config runs with an older version or workfile2? If so, what would happen?

Good question.

Running this config with an older version of workfiles2 will run fine because it will just ignore the new hook defined in the config.

However, running an old config with the new version of workfiles2 will run (without error) but will not have the new hook, which is responsible for fixing styling issues in Houdini with the new workfiles2 app. So to answer your question about engine requirement, the new version of workfiles2 should not be included in the config unless this change is included in the config.

Speaking of version requirements, that reminds me I will need to bump the tk-framework-qtwidgets version since that is a dependency.